_content/essays/four-dharma-traditions-of-tibet_mipham.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The Four Dharma Traditions of the Land of Tibet"
+authors:
+  - mipham
+translator: "Adam Pearcey"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/mipham/four-dharma-traditions-of-tibet"
+drive_links:
+  - "https://drive.google.com/file/d/1o0n3RCVuO4v3GttguakIh7NkM_knrl4k/view?usp=sharing"
+course: tibetan
+tags:
+  - view
```

Comment:
Why "right view"?

Reply:
Also, this should have gotten the `subcat: poetry` marker. No worries, I've gone ahead and made the changes.